controllers/user.js

```diff
@@ -1,14 +1,24 @@
 import Sequelize from 'sequelize';
 import jwt from 'jsonwebtoken';
 import 'babel-polyfill';
+import dotenv from 'dotenv';
+import sendmail from '../services';
 import models from '../models';
 
+dotenv.config();
 const { iLike, or } = Sequelize.Op;
+const { sendWelcomeMail } = sendmail;
 const { User, Profile, Article } = models;
 const secret = process.env.SECRET_KEY;
+
```

Comment:
If there must be a space here, it should be between lines 11 and 12 (since secret, time and generateToken) have to do with the same thing., not 12 and 14.